pytorch_lightning/strategies/ddp.py

```diff
@@ -80,7 +79,7 @@ class DDPStrategy(ParallelStrategy):
     devices (e.g. GPU) per node. It is very similar to how :mod:`torch.distributed.launch` launches processes.
     """
 
-    distributed_backend = _StrategyType.DDP
+    distributed_backend = "ddp"
```

Comment:
We should get rid of `distributed_backend` attribute in the future.

Reply:
With current PR, this `distributed_backend` only means strategy_name, which matches `ddp ` to DDPPlugin. Agree, we can rename and clean up `distributed_backend` after this PR, and rely on instance check. We have a issue for it #10821

Reply:
updated